pkgs/applications/audio/csound/csound-qt/default.nix

```diff
@@ -1,42 +1,40 @@
 { stdenv, csound, desktop-file-utils,
   fetchFromGitHub, python, python-qt, qmake,
-  qtwebengine, rtmidi, unzip }:
+  qtwebengine, qtxmlpatterns, rtmidi }:
 
 stdenv.mkDerivation rec {
   name = "csound-qt-${version}";
-  version = "0.9.6-beta2";
+  version = "0.9.6-beta3";
 
   src = fetchFromGitHub {
     owner = "CsoundQt";
     repo = "CsoundQt";
     rev = "${version}";
-    sha256 = "12jv7cvns3wj2npha0mvpn88kkkfsxsvhgzs2wrw04kbrvbhbffi";
+    sha256 = "007jhkh0k6qk52r77i067999dwdiimazix6ggp2hvyc4pj6n5dip";
   };
 
-  patches = [ ./rtmidipath.patch ];
+  patches = [ ./rtmidipath.patch ./examplepath.patch ];
 
-  nativeBuildInputs = [ qmake qtwebengine ];
+  nativeBuildInputs = [ qmake qtwebengine qtxmlpatterns ];
 
-  buildInputs = [ csound desktop-file-utils rtmidi unzip ];
+  buildInputs = [ csound desktop-file-utils rtmidi ];
 
   qmakeFlags = [ "qcs.pro" "CONFIG+=rtmidi" "CONFIG+=pythonqt"
+                 "CONFIG+=record_support" "CONFIG+=html_webengine"
                  "CSOUND_INCLUDE_DIR=${csound}/include/csound"
                  "CSOUND_LIBRARY_DIR=${csound}/lib"
                  "RTMIDI_DIR=${rtmidi.src}"
-                 "PYTHONQT_SRC_DIR=${python-qt}/lib"
+                 "PYTHONQT_SRC_DIR=${python-qt}/include/PythonQt"
                  "PYTHONQT_LIB_DIR=${python-qt}/lib"
-                 "LIBS+=${python-qt}/lib/libPythonQt-Qt5-Python2.7.so"
-                 "LIBS+=${python-qt}/lib/libPythonQt_QtAll-Qt5-Python2.7.so"
+                 "LIBS+=-L${python-qt}/lib"
                  "INCLUDEPATH+=${python-qt}/include/PythonQt"
                  "INCLUDEPATH+=${python}/include/python2.7"
-                 "INSTALL_DIR=$(out)"
-                 "SHARE_DIR=$(out)/share"
                  ];
 
-  installPhase = ''
-    mkdir -p $out
-    cp -r bin $out
-    make install
+  preConfigurePhases = ["configureWorkaroundPhase"];
+
+  configureWorkaroundPhase = ''
```

Comment:
It would be simpler to just use the standard `preConfigure` instead.

Reply:
I used @symphorien's method of just fixing it without any phase manipulation.